packages/plugin-dom-layout/src/ActionableDomObjectRenderer.ts

```diff
@@ -66,14 +66,15 @@ export class ActionableDomObjectRenderer extends NodeRenderer<DomObject> {
         const editor = this.engine.editor;
         const select = !!button.selected(editor);
         const enable = !!button.enabled(editor);
+        const visible = !!button.visible(editor);
 
         const attrSelected = element.getAttribute('aria-pressed');
         if (select.toString() !== attrSelected) {
             element.setAttribute('aria-pressed', select.toString());
             if (select) {
-                element.classList.add('pressed');
+                element.classList.add('active');
```

Comment:
Why ?

Reply:
Because it's a class used in Odoo.

Reply:
I don't think we should choose our class names because of Odoo. Especially when they are bad class names like this one.

Reply:
What is the last word about that one?

Reply:
?

Reply:
@dmo-odoo

Reply:
done